packages/wallets/src/features/cfd/components/CFDPlatformsListAccounts/CFDPlatformsListAccounts.tsx

```diff
@@ -37,16 +34,11 @@ const CFDPlatformsListAccounts: React.FC<TProps> = ({ onMT5PlatformListLoaded })
     const hasCTraderAccount = !!ctraderAccountsList?.length;
     const hasDxtradeAccount = !!dxtradeAccountsList?.length;
 
-    useEffect(() => {
-        onMT5PlatformListLoaded?.(isMT5FetchedAfterMount);
-        return () => onMT5PlatformListLoaded?.(false);
-    }, [isMT5FetchedAfterMount, onMT5PlatformListLoaded]);
-
     return (
         <div className='wallets-cfd-list-accounts__content'>
             {/* TODO: Update loader with updated skeleton loader design */}
-            {!isMT5FetchedAfterMount && <TradingAppCardLoader />}
-            {isMT5FetchedAfterMount &&
+            {(isMT5Loading || !isFetchedAfterMount) && <TradingAppCardLoader />}
+            {(!isMT5Loading || isFetchedAfterMount) &&
```

Comment:
Is there a reason for adding both conditions here? Isn't isFetchedAfterMount = !isLoading

Reply:
isLoading before mounting, isFetchedAfterMount happens after mounting as the name suggest.

I want to check both before and after mounting for these mt5 accounts